### ISSUE TITLE
Refusing to run, non stale lock file [/var/lock/citadel.lock] present

### ISSUE THREAD

**Issue by alprinsden** [ess/citadel#8]
i found that in the log

[ Sun Nov 29 21:06:02 2015 ] INFO: START_RUN; Ban_Period: [9999999] Allowed_Cons: [100]
[ Sun Nov 29 21:06:02 2015 ] INFO: END_RUN; Shutting down.
[ Sun Nov 29 21:12:08 2015 ] INFO: START_RUN; Ban_Period: [9999999] Allowed_Cons: [100]
[ Sun Nov 29 21:12:08 2015 ] INFO: START_RUN; Ban_Period: [9999999] Allowed_Cons: [100]
[ Sun Nov 29 21:12:08 2015 ] INFO: START_RUN; Ban_Period: [9999999] Allowed_Cons: [100]
[ Sun Nov 29 21:12:08 2015 ] INFO: START_RUN; Ban_Period: [9999999] Allowed_Cons: [100]
[ Sun Nov 29 21:12:08 2015 ] INFO: START_RUN; Ban_Period: [9999999] Allowed_Cons: [100]
[ Sun Nov 29 21:12:08 2015 ] CRIT: Refusing to run, non stale lock file [/var/lock/citadel.lock] present.
[ Sun Nov 29 21:12:08 2015 ] INFO: END_RUN; Shutting down.
[ Sun Nov 29 21:12:08 2015 ] INFO: END_RUN; Shutting down.
[ Sun Nov 29 21:12:08 2015 ] INFO: END_RUN; Shutting down.
[ Sun Nov 29 21:12:08 2015 ] INFO: END_RUN; Shutting down.
[ Sun Nov 29 21:12:08 2015 ] INFO: END_RUN; Shutting down.
[ Sun Nov 29 21:12:08 2015 ] INFO: END_RUN; Shutting down.
[ Sun Nov 29 21:13:01 2015 ] INFO: START_RUN; Ban_Period: [9999999] Allowed_Cons: [100]
[ Sun Nov 29 21:13:01 2015 ] INFO: END_RUN; Shutting down.

**Comment by sgsullivan** [comment]
Did you modify the citadel cron file at all? If so, can you show the contents of it? Based on the logs, it appears your system may either be heavily loaded, or the firewall command citadel is running is possibly hanging on your system. Can you also show the process output (ps faux) of the citadel pid, and any of its forked processes? For instance when citadel runs (say, csf), it will show csf under the main citadel process with a \_ like this

avahi     1980  0.0  0.0  32424  2480 ?        Ss   Oct29   0:00 avahi-daemon: running [data.local]
avahi     2070  0.0  0.0  32092    36 ?        S    Oct29   0:00  \_ avahi-daemon: chroot helper

Basically I am looking to see if one of the firewall commands citadel is running is hanging making its run times increase in length, which could then cause that type of log output. Though there being 5 START_RUN log entries at 21:12:08 makes me wonder if the citadel cron was modified somehow, or if you have multiple copies in /etc/cron.d ?

**Comment by sgsullivan** [comment]
I am closing this as there hasn't been any response to the inquiries.
